content/ko/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -0,0 +1,347 @@
+---
+title: 레플리카셋
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+레플리카셋의 목적은 레플리카 파드 셋이 주어진 시간동안 안정적인 실행 및 유지하는 것이다.
+이처럼 레플리카셋은 보통 동일한 파드에 대한 설정된 갯수의 가용성을 보증하는데 사용한다.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 레플리카셋의 작동 방식
+
+레플리카셋은 파드를 식별하는 셀렉터와 적절한 수의 파드를 유지하기 위한 레플리카의 수, 
```

Comment:
[원문]
A ReplicaSet is defined with fields, including a selector that specifies how to **identify Pods it can acquire**, a number
of replicas indicating **how many Pods it should be maintaining**, and **a pod template specifying the data** of new Pods
**it should create to meet the number of replicas criteria**.

원문을 고려했을 때, 누락되거나 다른 의미로 번역된 부분이 있어서 보완이 필요해보입니다.
아래 수정 예시 참고하시기 바랍니다.

```suggestion
레플리카셋을 정의하는 필드는 획득 가능한 파드를 식별하는 방법이 명시된 셀렉터, 유지해야 하는 파드 개수를 명시하는 레플리카의 개수, 
```